go/roothash/api/api.go

```diff
@@ -162,15 +162,23 @@ type ExecutionDiscrepancyDetectedEvent struct {
 type MergeDiscrepancyDetectedEvent struct {
 }
 
+// FinalizedEvent is a finalized event.
+type FinalizedEvent struct {
+	Value []byte `json:"value"`
+}
+
 // Event is a roothash event.
 type Event struct {
+	RuntimeID common.Namespace `json:"runtime_id"`
```

Comment:
These are most prob. breaking changes (will add a fragment), are we fine with including the runtime ID in events? 

Also the added `FinalizedEvent` i could probably work around changing these two things if we don't like ti.

cc @kostko

Reply:
Yeah I think it makes sense to include RuntimeID as the `GetEvents` endpoint returns events for all runtimes. While we are here, can we update all the internal Tendermint events emitted by the roothash service to include a "runtime ID" attribute (e.g., `KeyRuntimeID`)? This would allow us to only subscribe to events for specific runtimes.